ui/de-lib/src/main/java/org/iplantc/de/diskResource/client/presenters/metadata/MetadataPresenterImpl.java

```diff
@@ -116,5 +233,17 @@ public void onSuccess(MetadataTemplate result) {
     public DiskResource getSelectedResource() {
         return resource;
     }
+    
+    public static DiskResourceMetadata newMetadata(String attr, String value, String unit) {
```

Comment:
Could this be moved to a utility class or done differently so `DiskResourceMetadataViewImpl` doesn't have to call this method?